frank-doc-doclet/src/test/java/org/frankframework/frankdoc/IntegrationWithLogTest.java

```diff
@@ -79,7 +79,6 @@ public void testContradictionsWithDefaultAndMandatoryAreLogged() throws Exceptio
 			String actualXsd = docWriter.getSchema();
 			String expectedXsd = TestUtil.getTestFile("/doc/examplesExpected/attributeDefault.xsd");
 			TestUtil.assertEqualsIgnoreCRLF(expectedXsd, actualXsd);
-			appender.assertLogged("Attribute [Master.explicitNullOnPrimitive] is of primitive type [short] but has default value null");
```

Comment:
Zeker weten? LIjkt niets te maken te hebben met ibisdocref removal

Reply:
Het derde argument van `IbisDoc` documenteerd wat de default waarde is. Ik heb de annotatie weggehaald, dus nu heeft hij geen default waarde meer. Vandaar dat deze log statement niet meer van toepassing is.